supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -259,6 +259,29 @@ When combined with privileges, permissions define a user's or resource's overall
 
 *See also*:
 
+[discrete]
+[[plugin]]
+==== image:images/yes.png[yes] plugin (noun)
+*Description*: A "plugin" is a software component that adds new features or extends the functionality of an existing application or service. 
+In Red Hat, "plugin" is preferred over "plug-in". 
```

Comment:
```suggestion
In Red Hat product documentation, "plugin" is preferred over "plug-in". 
```
- Beginning of the the sentence needs something more than `In Red Hat`, imo.
- Also, is `plugin` just "preferred"? Or do we want to be more opinionated here? Maybe something like `Use "plugin" rather than "plug-in". To my ears, using`preferred` leaves it open to use either form, though "plug-in" is immediately listed as an "incorrect" form and we say: "Use it: no".

Reply:
@bburt-rh Thank you for reviewing :-) Great points. I have rephrased the sentence in line 266 to:

`In Red Hat product documentation, use "plugin" rather than "plug-in".`

Reply:
@michelle-purcell Or consider dropping the introductory phrase entirely. We're already in the SSG so we already know we're referring to RH product docs.

Reply:
@bburt-rh - +1 - The fluff is now removed!

Reply:
Do we need a separate entry for plug-in just to refer to plugin? 
Also, do we need to cover plug(-)in (adjective)?